libs/ardour/wscript

```diff
@@ -373,7 +373,7 @@ def build(bld):
                         'SNDFILE','SAMPLERATE','LRDF','AUDIOUNITS', 'GIOMM',
                         'OSX','BOOST','CURL','TAGLIB','VAMPSDK','VAMPHOSTSDK','RUBBERBAND']
     obj.use          = ['libpbd','libmidipp','libevoral',
-                        'libaudiographer',
+                        'libaudiographer', 'libexecinfo',
```

Comment:
`libexecinfo` should be probably be added as uselib "EXECINFO" or otherwise special cased for BSD.